data/card_catalogue/npc/npc-generic.json

```diff
@@ -1,30 +1,17 @@
 [
   {
     "title": "Quick Shot",
-    "upgrade": "Empty the Clip",
+    "upgrade": null,
     "tier": "blue",
     "details": {
       "rules": {
         "cast": "Make an instant attack that cannot miss.",
         "hand": null
       },
-      "flavour": "A single, well timed strike can have greater impact than any barrage of blows",
+      "flavour": "From way downtown",
```

Comment:
I feel like a `!` fits the vibe here

Reply:
for sure, added in https://github.com/elliottomlinson/cardmaster/pull/214/commits/2458099cb4795aa89805ae6ccfc209597a30a1c0